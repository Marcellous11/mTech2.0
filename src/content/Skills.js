import React from 'react';
import '../styles/Skills.css';
const Skills = () => {
	return (
		<div className="Skills">
			<h3 className="Title">Skills</h3>
			<div className="Skills" />
			<ul className="Skills-List">
				<h3 className="Skill-Sections">Languages </h3>
				<ul>
					<li>
						<p className="Skill-Title">JavaScript</p>
						<div className="Skills-Container">
							<div className="Skill-iso JavaScript" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">Python is the best</p>
						<div className="Skills-Container">
							<div className="Skill-iso Python" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">Node</p>
						<div className="Skills-Container">
							<div className="Skill-iso Node" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">SQL </p>
						<div className="Skills-Container">
							<div className="Skill-iso SQL" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">Java</p>
						<div className="Skills-Container">
							<div className="Skill-iso Java" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">C++</p>
						<div className="Skills-Container">
							<div className="Skill-iso C" />
						</div>
					</li>
				</ul>

				<h3 className="Skill-Sections">Libraries </h3>
				<ul>
					<li>
						<p className="Skill-Title">React</p>
						<div className="Skills-Container">
							<div className="Skill-iso React" />
						</div>
					</li>
				</ul>

				<h3 className="Skill-Sections">Framework</h3>
				<ul>
					<li>
						<p className="Skill-Title">Express</p>
						<div className="Skills-Container">
							<div className="Skill-iso Express" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">Flask</p>
						<div className="Skills-Container">
							<div className="Skill-iso Flask" />
						</div>
					</li>
				</ul>

				<h3 className="Skill-Sections">Other</h3>
				<ul>
					<li>
						<p className="Skill-Title">DOM Manipulation</p>
						<div className="Skills-Container">
							<div className="Skill-iso DOM" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">Object Oriented Programming</p>
						<div className="Skills-Container">
							<div className="Skill-iso Oriented" />
						</div>
					</li>

					<li>
						<p className="Skill-Title">PostgreSQL</p>
						<div className="Skills-Container">
							<div className="Skill-iso PostgreSQL" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">Git</p>
						<div className="Skills-Container">
							<div className="Skill-iso Git" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">Ajax</p>
						<div className="Skills-Container">
							<div className="Skill-iso Ajax" />
						</div>
					</li>
					<li>
						<p className="Skill-Title">jQuery</p>
						<div className="Skills-Container">
							<div className="Skill-iso jQuery" />
						</div>
					</li>
				</ul>
			</ul>
		</div>
	);
};
export default Skills;
