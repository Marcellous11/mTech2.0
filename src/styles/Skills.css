.Skill-Title {
	color: rgb(255, 255, 255);
	font-weight: 700;
}
.Skills-List {
	list-style: none;
}
.Skills-List ul {
	color: white;
	list-style-type: decimal-leading-zero;
}
.Skills-Container {
	width: 70%;
	height: .25em;
	background-color: rgb(151, 151, 151);
}
.Skill-iso {
	height: .25em;
}

.JavaScript {
	width: 97%;
	background-color: rgb(55, 255, 0);
}
.DOM {
	width: 97%;
	background-color: rgb(55, 255, 0);
}
.React {
	width: 97%;
	background-color: rgb(55, 255, 0);
}
.Node {
	width: 90%;
	background-color: rgb(111, 255, 0);
}
.Express {
	width: 97%;
	background-color: rgb(55, 255, 0);
}
.PostgreSQL {
	width: 95%;
	background-color: rgb(111, 255, 0);
}
.SQL {
	width: 87%;
	background-color: rgb(162, 255, 0);
}
.Git {
	width: 90%;
	background-color: rgb(162, 255, 0);
}
.Ajax {
	width: 85%;
	background-color: rgb(187, 255, 0);
}
.jQuery {
	width: 80%;
	background-color: rgb(204, 255, 0);
}
.Python {
	width: 95%;
	background-color: rgb(132, 255, 0);
}
.Flask {
	width: 90%;
	background-color: rgb(89, 255, 0);
}
.Oriented {
	width: 96%;
	background-color: rgb(55, 255, 0);
}

.C {
	width: 65%;
	background-color: rgb(204, 255, 0);
}
.Java {
	width: 75%;
	background-color: rgb(200, 255, 0);
}

.Skill-Sections {
	color: rgb(255, 86, 227);
}

@media only screen and (max-width: 830px) {
	.Skills-List {
		margin-left: -1.5em;
	}
	.Skills-Container {
		width: 90%;
	}
}
