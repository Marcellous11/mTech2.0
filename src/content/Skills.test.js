import { logRoles, prettyDOM, render, screen } from '@testing-library/react';
import Skills from './Skills';

test('Smoke test, does the component render', () => {
	render(<Skills />);
});

test('Snapshot ', () => {
	const { asFragment } = render(<Skills />);

	expect(asFragment()).toMatchSnapshot();
});

test('JavaScript should be in the document', () => {
	const { getByText } = render(<Skills />);

	expect(getByText(`JavaScript`)).toBeInTheDocument();
});
test('Python should be in the document', () => {
	const { getByText } = render(<Skills />);

	expect(getByText(`Python`)).toBeInTheDocument();
});
test('SQL should be in the document', () => {
	const { getByText } = render(<Skills />);

	expect(getByText(`SQL`)).toBeInTheDocument();
});
test('C++ should be in the document', () => {
	const { getByText } = render(<Skills />);

	expect(getByText(`C++`)).toBeInTheDocument();
});
test('JavaScript should be in the document', () => {
	const { getByText } = render(<Skills />);

	expect(getByText(`Node`)).toBeInTheDocument();
});
test('JavaScript should be in the document', () => {
	const { getByText } = render(<Skills />);

	expect(getByText(`Node`)).toBeInTheDocument();
});
