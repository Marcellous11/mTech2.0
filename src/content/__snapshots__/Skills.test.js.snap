// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot  1`] = `
<DocumentFragment>
  <div
    class="Skills"
  >
    <h3
      class="Title"
    >
      Skills
    </h3>
    <div
      class="Skills"
    />
    <ul
      class="Skills-List"
    >
      <h3
        class="Skill-Sections"
      >
        langues
      </h3>
      <ul>
        <li>
          <p
            class="Skill-Title"
          >
            JavaScript
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso JavaScript"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            Python
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso Python"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            Node
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso Node"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            SQL 
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso SQL"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            Java
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso Java"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            C++
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso C"
            />
          </div>
        </li>
      </ul>
      <h3
        class="Skill-Sections"
      >
        Libaraies
      </h3>
      <ul>
        <li>
          <p
            class="Skill-Title"
          >
            React
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso React"
            />
          </div>
        </li>
      </ul>
      <h3
        class="Skill-Sections"
      >
        framwork
      </h3>
      <ul>
        <li>
          <p
            class="Skill-Title"
          >
            Express
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso Express"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            Flask
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso Flask"
            />
          </div>
        </li>
      </ul>
      <h3
        class="Skill-Sections"
      >
        Other
      </h3>
      <ul>
        <li>
          <p
            class="Skill-Title"
          >
            DOM Manipulation
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso DOM"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            Object Oriented Programming
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso Oriented"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            PostgreSQL
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso PostgreSQL"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            Git
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso Git"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            Ajax 
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso Ajax"
            />
          </div>
        </li>
        <li>
          <p
            class="Skill-Title"
          >
            jQuery
          </p>
          <div
            class="Skills-Container"
          >
            <div
              class="Skill-iso jQuery"
            />
          </div>
        </li>
      </ul>
    </ul>
  </div>
</DocumentFragment>
`;
